connect/src/protocols/cctp/cctpTransfer.ts

```diff
@@ -209,7 +209,7 @@ export class CircleTransfer<N extends Network = Network>
   }
 
   // init from source tx hash
-  private static async fromTransaction<N extends Network>(
+  static async fromTransaction<N extends Network>(
```

Comment:
why make these public and not just call the `from` method which should invoke this one?

Reply:
just made this change :+1: